src/ol_infrastructure/components/aws/olvpc.py

```diff
@@ -234,7 +229,7 @@ def __init__(
         :type opts: ResourceOptions
         """
         super().__init__('ol:infrastructure:aws:VPCPeeringConnection', vpc_peer_name, None, opts)
-        resource_options = ResourceOptions(parent=self).merge(opts)
+        resource_options = ResourceOptions.merge(ResourceOptions(parent=self), opts)  # type: ignore
```

Comment:
Don't think this is making sense to me

Reply:
That was just attempting to address a MyPy complaint. The two versions are functionally equivalent.